core/src/toga/app.py

```diff
@@ -391,6 +391,16 @@ def home_page(self) -> str | None:
         menu items."""
         return self._home_page
 
+    @property
+    def dark_mode(self) -> bool | None:
+        """Whether the user has dark mode enabled in their environment (read-only).
+
+        Returns None if we cannot determine dark mode settings."""
```

Comment:
We can use specific markup for the return value.
```suggestion
        :returns: A boolean describing if the app is in dark mode; ``None`` if Toga
        cannot determine if the app is in dark mode.
        """
```